db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_03_02_154436) do
+ActiveRecord::Schema.define(version: 2020_05_21_150645) do
```

Comment:
why is this file changing?

Reply:
Running db:migrate for the data migration causes the version to update in this file.